sidebars.js

```diff
@@ -162,6 +162,7 @@ module.exports = {
 
       ],
     },
+    "web-wallet",
```

Comment:
We should keep the options on both menus synced (except Introduction, Quick Start Demo and Releases):

<img width="381" alt="image" src="https://github.com/0xPolygonID/docs/assets/15896177/33033d53-2287-4895-b5f4-2cf6387d87dc">

This means:

* Adding Web Wallet also to the top menu
* Moving JS-SDK below smart contracts in the top menu (or the other way around in the side menu)

Reply:
Yes i had written this down for discussion cauz we need to decide on whether to have web wallet in 'Tools' or 'Build' technically as it is a tool it should be placed in tools. The same was suggested by @elias-garcia as well.
But i noticed that all the tools are pointing to an external url where people can use the tool, the same is not possible with Web Wallet- shoud we point it to its documentation istelf?

Reply:
Well, these are different discussions. My previous comment applies to whatever we decide to do regarding tools. In Build, we need to point to the same documentation page we point from the sidebar.

> the same is not possible with Web Wallet

Why is it not possible? I think of the demo app as the entry point to the wallet.

Reply:
Got it @amonsosanz , I will do as follows:
1) sync the content in build and sidebar
2) Add wallet web in tools and point it to the demo

Reply:
Done

Reply:
I'm still missing the option Web Wallet in the Build top menu.

Reply:
Have added it in the Tools Top menu

Reply:
Yep, I've seen it. Now it's only missing in the Build top menu.

Reply:
> Should we add it in both- tools and build?

I'd say yes.

  * In Build, we add a link to the internal docs page, as we have on the left sidebar (we want to keep the left sidebar and the Build menu synced). T
  * In Tools we add an external link to the tool, as we do with every other web tool we have.

Reply:
True